### DISCUSSION TITLE
multitple prompts - 'end' image?

### DISCUSSION THREAD

**Discussion by telemole** [Extraltodeus/advanced-loopback-for-sd-webui#2] [Q&A]
Hi I LOVE this tool!!

I want to use the multple prompts feature to change prompt throughout the generation but the readme refers to the end image : 

"The multiple prompts option will switch prompt after the "end" imagee has been reached (refering to the input box).

So 0-10 will switch prompt every 10 images."

where do I enter this 0-10? how many prompts can be used? thanks so much for any help!!

**Comment by Extraltodeus** [comment]
Thank you!

I refer to the Denoising strength start/end text boxes which will define when the prompt will change.

You can use as many prompts as you want.
